src/frontend/src/lib/components/hero/HeroSignIn.svelte

```diff
@@ -1,15 +1,45 @@
 <script lang="ts">
+	import type { ComponentType } from 'svelte';
+	import IconGlasses from '$lib/components/icons/IconGlasses.svelte';
+	import IconInfinity from '$lib/components/icons/IconInfinity.svelte';
+	import IconShieldAlert from '$lib/components/icons/IconShieldAlert.svelte';
 	import ButtonAuthenticate from '$lib/components/ui/ButtonAuthenticate.svelte';
 	import { signIn } from '$lib/services/auth.services';
 	import { i18n } from '$lib/stores/i18n.store';
+
+	const infoList: { label: string; icon: ComponentType }[] = [
```

Comment:
This only works as long as we only supports english.

For example turn `infoList` into a `let` or other alternatives.

Reply:
right!